pkg/rpc/frontend/onboarding-stage-impl.go

```diff
@@ -23,3 +26,42 @@ func (s *Server) GetCurrentOnboardingStage(
 		CurrentStage: enums.OnboardingStage_ONBOARDING_STAGE_EMAIL_VERIFICATION,
 	}, nil
 }
+
+// OTP storage (in-memory for demonstration purposes)
+var otpStore = map[string]string{}
+
+func (s *Server) VerifyAadhar(ctx context.Context, req *onboarding.AadharVerificationRequest) (*onboarding.AadharVerificationResponse, error) {
+	if req.Otp == "" {
+		// Generate OTP
+		otp := fmt.Sprintf("%06d", rand.Intn(1000000))
```

Comment:
check if error is getting returned here , if so handle with care

Reply:
![image](https://github.com/user-attachments/assets/b8b8e756-68ab-40c8-b529-b6a06b0822de)
The return type is integer only so no error is getting returned here